config/management/src/genesis.rs

```diff
@@ -33,7 +33,12 @@ impl Genesis {
         let genesis = vm_genesis::encode_genesis_transaction_with_validator(
             association_key,
             &validators,
+<<<<<<< HEAD
```

Comment:
this is certainly an interesting change... ;)

Reply:
I have my reasons...